mina-archive/scripts/archive-readiness.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# This script checks if the server port is open and listening,
+# by any process, without installing additional tooling
+
+# The /proc/*/net/tcp prints the network connections/sockets with hexadecimal values
+# The `00000000` is the `0.0.0.0` ip address in hexa format.
+# The `%04X` will print the port in hexa format.
+
+grep "$(printf '00000000:%04X' "$ARCHIVE_SERVER_PORT_RPC")" /proc/*/net/tcp
```

Comment:
praise: in this toolless docker world I really like this way of checking it! given grep is in the container 🤣